### ISSUE TITLE
standard template size for svg markers

### ISSUE THREAD

**Issue by antoniolocandro** [OpenAviationMap/rendering#1]
I have checked the SVG markers and they don´t have a `standard size`, I think it would be a good idea to create an iconset that can be share also with GIS software like QGIS[http://www.qgis.org]. I would suggest that we recreate the whole set of icons as per ICAO documentation

**Comment by akosmaroy** [comment]
sounds like a good idea. would you be willing to update the SVGs / create more SVGs according to the ICAO recommendation?

**Comment by antoniolocandro** [comment]
Sure, is there a reason for `SVG` files to be in root with the `SLD` files, may I suggest we put them in their own folder and further subdivide SVG into the different sub sets e.g.

**SLD**

**SVG**
- Aerodromes
- Navaids
- Waypoints
- ATS
- Obstacles
- Visual Aids
- Miscellaneous
- Symbols for Aerodrome Charts
- Symbols for Obstacle charts

Also the symbols have a _ID_ number that precedes it in _Annex 4 Appendix 2. ICAO CHART SYMBOLS_

**Comment by akosmaroy** [comment]
yes, we could do that!

**Comment by antoniolocandro** [comment]
OK, I am thinking we could publish that we are recreating ICAO symbols in SVG in the AIS AGORA and AIXM forum to generate some buzz maybe some other persons there might contribute to creating the symbols. What do you think? I know many people start making their own symbols so if we can create a free repository that any software could use that would be great

**Comment by akosmaroy** [comment]
yes, it would be excellent to have a free repository, with multiple commiters

**Comment by molysgaard** [comment]
I really like this idea @antoniolocandro! Icons/symbols are an essential part of a good map and the OAM project.
